#VRML_SIM R2022a utf8
WorldInfo {
}
Viewpoint {
  orientation -0.5773500853651613 0.5773500920254214 0.577350630178125 2.0943994423762824
  position 1.696965368158665e-05 1.6969701666535997e-05 6.001368646922227
}
TexturedBackground {
}
TexturedBackgroundLight {
}
PointLight {
  attenuation 0 0 1
  intensity 21
  location 4 4 -1
  radius 10
  castShadows TRUE
}
RectangleArena {
  floorSize 1.5 0.25
}
DEF epuck E-puck {
  translation -0.625 0 -6.39627e-05
  rotation 0 0 1 0
  controller "task1"
  supervisor TRUE
  turretSlot [
    DEF Front DistanceSensor {
      translation 0.045 0 -0.011
      name "front distance sensor"
      lookupTable [
        0 0 0
        1.27 1.27 0.1
      ]
    }
    DEF Right DistanceSensor {
      translation 0.01 -0.035 -0.011
      rotation 0 0 1 -1.5707853071795865
      name "right distance sensor"
      lookupTable [
        0 0 0
        1.27 1.27 0.1
      ]
    }
    DEF Rear DistanceSensor {
      translation -0.025 0 -0.011
      rotation 0 0 1 3.1415
      name "rear distance sensor"
      lookupTable [
        0 0 0
        1.27 1.27 0.1
      ]
    }
    DEF Left DistanceSensor {
      translation 0.01 0.035 -0.011
      rotation 0 0 1 1.5708
      name "left distance sensor"
      recognitionColors [
        0 0 0
      ]
      lookupTable [
        0 0 0
        1.27 1.27 0.1
      ]
    }
    InertialUnit {
      rotation -1 0 0 -5.307179586466759e-06
    }
    Camera {
      translation 0.043 0 -0.015
      name "camera1"
      recognitionColors [
        1 1 0
      ]
      fieldOfView 1.57
      width 80
      height 80
      recognition Recognition {
      }
    }
  ]
}
